### PR TITLE
[WIP] Update patelgp-add-caption's slide

### PR DESCRIPTION
**Pull request by patelgp**
## Changes 
Caption changed  

cc: @githubteacher

### CODE REVIEW

**Review by eostrovski — APPROVED** [review]
Looks good!

File: _posts/0000-01-11-patelgp.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Making test change
```
Comment (eostrovski) [inline _posts/0000-01-11-patelgp.md]: grate caption, not descriptive enough

**Review by eostrovski — CHANGES_REQUESTED** [review]
Fix your code!

**Review by eostrovski — APPROVED** [review]

**Comment by patelgp** [comment]
Merge

**Comment by patelgp** [comment]
merge

**Pull request merged**